Kakaotalk/UIConponents/TabViewSection.swift

```diff
@@ -15,31 +15,31 @@ struct TabViewSection: View {
                 MainView()
             }
             .tabItem {
-                Label("", systemImage: "person")
+                Label("Main", systemImage: "person")
                 
             }
             
             NavigationView {
                 ComunicationView()
             }
             .tabItem {
-                Label("", systemImage: "message")
+                Label("Chat", systemImage: "message")
                 
             }
             
             NavigationView {
                 PhoneView()
             }
             .tabItem {
-                Label("", systemImage: "phone")
+                Label("Phone", systemImage: "phone")
                 
             }
             
             NavigationView {
                 SettingPage()
             }
             .tabItem {
-                Label("", systemImage: "ellipsis")
+                Label("Setting", systemImage: "ellipsis")
```

Comment:
it would be better if we name this as `Settings`.

Reply:
Ok.